AlicerceTests/Extensions/XCTestCase.swift

```diff
@@ -20,8 +20,9 @@ extension XCTestCase {
 
 extension XCTestExpectation {
     func fulfillAfter(_ time: TimeInterval) {
```

Comment:
I also don't understand the use case for this timed `fulfil`. I think its best practice to invoke `fulfill` when the async operation has completed, not after some arbitrary time.

In my opinion, it makes tests brittle and can lead to bad habits of guessing the times operations take to complete.